packages/extension/app/background.js

```diff
@@ -205,6 +209,21 @@ const automation = {
     }).then(fn)
   },
 
+  resetBrowserState (fn) {
+    // We remove browser data. This is how we remove browser state for firefox
+    // (https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/browsingData/DataTypeSet).
```

Comment:
Nit: this comment sounds like we are only removing browser data for firefox but it seems like it's only linking to the firefox docs? Are there other docs we need to link or does this need reworded?

Reply:
I can update the comment, but there are multiple ways that we remove browser state. firefox is done here. chrome is done in cdpautomation

Reply:
Comment updated: https://github.com/cypress-io/cypress/commit/f3e5a748f3536133e4657cf8cb60ce0e9ec8c07a